op-devstack/sysgo/superroot.go

```diff
@@ -233,31 +233,30 @@ func getInteropCannonAbsolutePrestate(t devtest.CommonT) common.Hash {
 }
 
 func getInteropCannonKonaAbsolutePrestate(t devtest.CommonT) common.Hash {
-	return common.HexToHash(findKonaVariable(t, "KONA_INTEROP_PRESTATE_HASH"))
+	return loadKonaVersions(t, "KONA_INTEROP_PRESTATE_HASH").InteropPrestateHash
 }
 
 func getCannonKonaAbsolutePrestate(t devtest.CommonT) common.Hash {
-	return common.HexToHash(findKonaVariable(t, "KONA_PRESTATE_HASH"))
+	return loadKonaVersions(t, "KONA_PRESTATE_HASH").PrestateHash
 }
 
-func findKonaVariable(t devtest.CommonT, name string) string {
-	konaJustfilePath := "kona/justfile"
-	root, err := findMonorepoRoot(konaJustfilePath)
+func loadKonaVersions(t devtest.CommonT, name string) konaVersions {
```

Comment:
`name` is no longer used:
```suggestion
func loadKonaVersions(t devtest.CommonT) konaVersions {
```